_data/simulations/Linear_Elasticity_4a_update/meta.yaml

```diff
@@ -0,0 +1,88 @@
+_id: 69386760-beab-11e9-9786-cb6ab85cdbfe
+metadata:
+  author:
+    first: Shubhajit
+    last: Mondal
+    email: sm44@iitbbs.ac.in
+    github_id: smondal44
+  timestamp: ''
```

Comment:
You must provide a timestamp.

Reply:
@tkphd Do it has to be uploaded again?

Reply:
@tkphd Is it possible to put the timestamp in .yaml file?